docs/android/v2/release-notes/release-notes.mdx

```diff
@@ -19,6 +19,20 @@ import AndroidPrebuiltVersionShield from '@/common/android-prebuilt-version-shie
 | live.100ms:virtual-background: |<AndroidSdkVersionShield />|
 | live.100ms:hms-noise-cancellation-android: | <AndroidSdkVersionShield /> |
 
+## v2.9.74 - 2025-03-3
+### Added
+* Name space changes added 
+### Breaking 
+* Build might break, please migrate imports from `org.webrtc` to `hms.webrtc`
```

Comment:
Due to the namespace changes , webrtc files imports needs to be changed from `org.webrtc` to `hms.webrtc` where-ever present in your application